rclcpp/src/rclcpp/logger.cpp

```diff
@@ -64,4 +66,15 @@ Logger::set_level(Level level)
   }
 }
 
+rcpputils::fs::path
+Logger::get_logging_directory() const
+{
+  char * log_dir = NULL;
+  auto allocator = rcutils_get_default_allocator();
+  rcl_logging_get_logging_directory(allocator, &log_dir);
```

Comment:
Consider throwing an exception if this call fails.

Reply:
Thanks, done in https://github.com/ros2/rclcpp/pull/1509/commits/1b6c60f52776d92f7966bd4210138cea76067333.